guide/Radiance_support.rst

```diff
@@ -1,34 +1,146 @@
 Introduction to DART’s support for RTTOV
```

Comment:
```suggestion
.. index:: radiance

Introduction to DART’s support for RTTOV
```

I think it would be good to add 'radiance' so this pages shows up as the first result in docs.dart.ucar.edu when people search radiance